lib/routes/home/widgets/payments_list/payment_details_dialog.dart

```diff
@@ -36,10 +36,10 @@ Future<void> showPaymentDetailsDialog(
             mainAxisSize: MainAxisSize.min,
             crossAxisAlignment: CrossAxisAlignment.stretch,
             children: [
-              PaymentDetailsDialogDescription(
```

Comment:
I don't think the description widget should be removed, that was not what I had in mind. I was suggesting we use the extractTitle method you introduced on title widget and display the full description on description widget.

Reply:
I just added the subtitle back; it is showing the same text as the title as it is coming from the same property, the only differences are the style and the ellipses/multiline strategy to long texts.